src/components/forms/MemoryLimitAvailable.tsx

```diff
@@ -1,59 +1,153 @@
+    return null;
   }
 
-  const getAvailableMemory = () => {
-    if (!project?.config["limits.memory"]) {
-      return resources?.memory.total;
+  const getAvailableMemory = (resources: LxdResources[]) => {
+    const sortedResourcesMemories = resources
+      .map((r) => r.memory.total)
+      .sort((a, b) => a - b);
+    const minResourceMemory = sortedResourcesMemories[0];
+    const maxResourceMemory =
+      sortedResourcesMemories[sortedResourcesMemories.length - 1];
+    if (!projectResourceMemory) {
+      return {
+        range: [minResourceMemory, maxResourceMemory],
+        source: isClustered && currentServer ? "cluster member" : "",
+      };
     }
-    if (!resources?.memory.total) {
-      return limitToBytes(project.config["limits.memory"]);
+    const projectLimit = Number(projectResourceMemory);
+
+    if (sortedResourcesMemories.length === 0) {
+      return {
+        range: [projectLimit, projectLimit],
+        source: "project",
+      };
     }
-    return Math.min(
-      resources.memory.total,
-      Number(limitToBytes(project.config["limits.memory"])),
-    );
+    return {
+      range: [
+        Math.min(minResourceMemory, projectLimit),
+        Math.min(maxResourceMemory, projectLimit),
+      ],
+      source: "",
+    };
   };
 
-  const maxMemory = getAvailableMemory();
+  const unClusteredResources = unClusteredResource ? [unClusteredResource] : [];
+  const {
+    range: [minMemory, maxMemory],
+    source,
+  } = getAvailableMemory(
+    isClustered ? clusteredResources : unClusteredResources,
+  );
 
-  return maxMemory ? (
+  let resourceLink = null;
+  if (source === "project") {
+    resourceLink = (
+      <ResourceLink
+        type="project"
+        value={project?.name || "default"}
+        to={`/ui/projects/${encodeURIComponent(project?.name || "default")}`}
+      />
+    );
+  } else if (source === "cluster member" && currentServer) {
+    resourceLink = (
+      <ResourceLink
+        type="cluster-member"
+        value={currentServer}
+        to={`/ui/cluster/member/${encodeURIComponent(currentServer)}`}
+      />
+    );
+  }
+  const showHelpIcon = isClustered && minMemory !== maxMemory;
+
+  return (
     <>
-      Total memory: <b>{humanFileSize(maxMemory)}</b>
+      Total memory:{" "}
+      <b>
+        {humanFileSize(minMemory)}
+        {minMemory !== maxMemory && ` - ${humanFileSize(maxMemory)}`}
+        {showHelpIcon && (
+          <>
+            {" "}
+            <Icon
+              name="information"
+              className="help-link-icon"
+              title={`The available memory depends on the member that the instance will be created on. To determine available memory exactly, Select the "Cluster Member" option from the "Target" dropdown and choose a member.`}
+            />
+          </>
+        )}
+      </b>
+      {resourceLink && (
+        <>
+          {" "}
+          on {source} {resourceLink}
```

Comment:
This should probably be "in project (foo)" and "on cluster member (bar)". But not "on project (foo)" as now..